tuf-spec.md

```diff
@@ -298,14 +298,14 @@ Version: **1.0 (Draft)**
   delegated role is only trusted for some of the target files that the
   delegating role is trusted for.
 
-      Delegated developer roles can further delegate trust to other delegated
+      Delegated roles can further delegate trust to other delegated
   roles.  This provides for multiple levels of trust delegation where each
   role can delegate full or partial trust for the target files they are
   trusted for.  The delegating role in these cases is still trusted.  That is,
   a role does not become untrusted when it has delegated trust.
 
       Delegated trust can be revoked at any time by the delegating role signing
-  new metadata that indicates the delegated role is no longer trusted.
+  new metadata that no longer includes that delegation.
```

Comment:
Something feels off with the modified sentence.  Do you find the following revision easier to understand?

> The Targets role, or a delegated role, can revoke trust by signing new metadata that no longer includes that delegation.

Reply:
You're right: it's a little awkward. The replacement is ambiguous at "or a delegated role", unfortunately. (It's not clear if "delegated role" is the delegatee or just any non-Targets role --
-- which is what you mean.) Perhaps "or any delegating role", so that it doesn't sound like it's the delegatee.

Or, how about:
`Any delegation can be revoked at any time: the delegating role need only sign new metadata that no longer contains that delegation.`

Maybe we can add this to that if you think it is needed:
`The current state of trust is always represented by the most recent version of metadata.`

Reply:
"Any delegation can be revoked at any time: the delegating role need only sign new metadata that no longer contains that delegation."

That's clear to me.

Reply:
K, I'll amend the commit to that instead, then (and update last modified).